chapters/en/slides/module3/module3_02.md

```diff
@@ -33,13 +38,16 @@ Under the hood, Git only saves the changes between your versions, so it is also
 
 <img src='/module3/vc-local-github.png' width="55%" alt="404 image" />
 
-<center/>
+</center>
+
+Notes:
+Both copies of the repository have a **working directory** where you can create, store, edit, and delete files (e.g., `analysis.ipynb` in the image). 
```

Comment:
To avoid starting the 2 paragraphs with "Both copies of the repository have..." we can use bullet points.

Both copies of the repository have:
* a working directory
* the full project history